app/src/main/java/org/jellyfin/androidtv/ui/shared/StrokeTextView.kt

```diff
@@ -15,6 +21,7 @@ class StrokeTextView @JvmOverloads constructor(
 ) : AppCompatTextView(context, attrs, defStyleAttr) {
 	var strokeWidth = 0.0f
 	private var isDrawing: Boolean = false
+	private val subtitlesTextColor = get<UserPreferences>(UserPreferences::class.java)[UserPreferences.subtitlesTextColor]
```

Comment:
```suggestion
	private val subtitlesTextColor = get<UserPreferences>()[UserPreferences.subtitlesTextColor]
```
(might need to update the import)

Reply:
Unfortunately the above code does not compile, would you have another suggestion to get this to compile?

Reply:
Add the KoinComponent interface

Reply:
Thanks that did the trick